Mathlib/Algebra/Polynomial/Roots.lean

```diff
@@ -310,6 +310,14 @@ theorem mem_nthRootsFinset {n : ℕ} (h : 0 < n) {x : R} :
 @[simp]
 theorem nthRootsFinset_zero : nthRootsFinset 0 R = ∅ := by classical simp [nthRootsFinset_def]
 
+theorem mem_nthRootsFinset_map {S F : Type*} [CommRing S] [IsDomain S] [FunLike F R S]
```

Comment:
This name suggests the mapping is happening to `nthRootsFinset`. Following [`map_mem_nonZeroDivisors`](https://leanprover-community.github.io/mathlib4_docs/Mathlib/Algebra/GroupWithZero/NonZeroDivisors.html#map_mem_nonZeroDivisors) I propose:
```suggestion
theorem map_mem_nthRootsFinset {S F : Type*} [CommRing S] [IsDomain S] [FunLike F R S]
```